Source/Text/SvgTextBase.Drawing.cs

```diff
@@ -22,8 +22,9 @@ public override RectangleF Bounds
                 {
                     //When empty Text span, don't add path
```

Comment:
Please change this comment as well.
(If string is blank, GraphicsPath points will be empty.)